src/styles/common-style.ts

```diff
@@ -113,3 +113,36 @@ export const borderBox = css`
     box-sizing: border-box;
   }
 `;
+
+export const hoverShortTransitionTiming = css`
+  :hover {
+    transition: 0.1s;
+  }
+
+  :not(:hover) {
+    transition: 0.1s;
+  }
```

Comment:
I think this should give the same behavior:
```suggestion
  transition: 0.1s;
```